addons/hr_timesheet/models/hr_employee.py

```diff
@@ -9,4 +9,6 @@ class HrEmployee(models.Model):
 
     timesheet_cost = fields.Monetary('Timesheet Cost', currency_field='currency_id',
     	groups="hr.group_hr_user", default=0.0)
+    overtime_cost = fields.Monetary('Overtime Cost', currency_field='currency_id',
+        groups="hr.group_hr_user", default=0.0)
```

Comment:
currency_field and default are actually the default values. It can be removed (on the other fields too while we're at it).
```suggestion
    overtime_cost = fields.Monetary('Overtime Cost', groups="hr.group_hr_user")
```